package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "lint-fix": "eslint --fix ./ --ext .js --color",
```

Comment:
Please change the script to lint:fix

Reply:
Thank you for your feedback. This has been corrected